Cargo.toml

```diff
@@ -10,8 +10,8 @@ edition = "2018"
 kelvin = "0.18"
 nstack = "0.4"
 lazy_static = "1.3.0"
-hades252 = { git = "https://github.com/dusk-network/hades252", version = "0.6.1" }
-dusk-plonk = "0.2.6"
+hades252 = { git = "https://github.com/dusk-network/hades252", version = "0.7.0" }
```

Comment:
I think you meant `tag` instead of `version` (since `hades252` is not on crates.io).

Reply:
When you import from git, cargo directly understands `version` as the `release` and `tag` as the `tag`. So It is correct, that's why the tests are passing anyway. 
I can put the tag if you want to.

Reply:
Yep, My bad. Fixed!